functions/src/util.ts

```diff
@@ -57,13 +57,17 @@ export function getChangedDocumentId(change: Change<DocumentSnapshot>): string {
  * @return {string[]} An array of fields.
  */
 export function getFieldsToIndex(): string[] {
-  return config.fieldsToIndex ? config.fieldsToIndex.split(/[ ,]+/) : []
+  return config.meilisearch.fieldsToIndex
+    ? config.meilisearch.fieldsToIndex.split(/[ ,]+/)
+    : []
```

Comment:
Where these not validated previously?

Reply:
There are not validated there are formated so I can use them more easily. Where do you want to validate them?

Reply:
Yes be before we arrive here, were they validated by your validation functions?

Reply:
Only if you use it through the script, but this function is also used by the extension through (Firebase CloudFunction).